plutus-core/cost-model/create-cost-model/BuiltinMemoryModels.hs

```diff
@@ -110,6 +110,8 @@ builtinMemoryModels = BuiltinCostModelBase
   , paramHeadList                        = Id $ ModelOneArgumentConstantCost    32
   , paramTailList                        = Id $ ModelOneArgumentConstantCost    32
   , paramNullList                        = Id $ ModelOneArgumentConstantCost    32
+  , paramLengthArray                     = Id $ ModelOneArgumentConstantCost    99
+  , paramListToArray                     = Id $ ModelOneArgumentConstantCost    99
```

Comment:
Dummy values

Reply:
> @kwxm is this what should be done here before costing? Particularly given that listToArray isn't gonna be constant-cost.

Good question.  I think that this doesn't really matter **as long as you remember to fix it**.  The only time this gets used is when you run `generate-cost-model`, which uses this information to fill in the memory function in the JSON version of the cost model, so it has to be correct before you generate the final form of the cost model.